contracts/Claims.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.7.5;
+
+// @notice Used to create Radicle Link identity claims
+contract Claims {
+    /// @notice Emitted on every Radicle Link identity claim
+    /// @param addr The account address which made the claim
+    event Claimed(address indexed addr);
+
+    /// @notice Creates a new claim of a Radicle Link identity.
+    /// Every new claim invalidates previous ones made with the same account.
+    /// The claims have no expiration date and don't need to be renewed.
+    /// If either `version` is unsupported or `payload` is malformed as per `version`,
+    /// the previous claim is revoked, but a new one isn't created.
+    /// Don't send a malformed transactions on purpose, to properly revoke a claim see `version`.
```

Comment:
This should say "see root"?

Reply:
It's a reference to a parameter, now it says ``see `format` ``, is it ok?